gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+              <NxTable.Cell>
+                This is the title of the list.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Item</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This is the standard list element that is not clickable, does not respond to hover events, and
+                does not have selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Button</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This list element that is clickable, responds to hover events, and has selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Link</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This list element behaves as a link, i.e. an <NxCode>{'<a>'}</NxCode> tag, and requires
+                a <NxCode>{'href'}</NxCode> prop. This element is also clickable, responds to hover events,
+                and has selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Text</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode>, <NxCode>NxList.Button</NxCode>,
+                or <NxCode>NxList.Link</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                The primary text content of the list item, displayed in a heavier font weight.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Subtext</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode>, <NxCode>NxList.Button</NxCode>,
+                or <NxCode>NxList.Link</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                The secondary text content of the list item displayed as non-bolded text.
+                The subtext is displayed below the bolded, main list item text.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Actions</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode></NxTable.Cell>
+              <NxTable.Cell>A container for buttons inside list items. Use this when you want to have
+                a button/buttons on the far right.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Empty</NxCode></NxTable.Cell>
```

Comment:
I doubt we want this to be an explicit component but rather have the empty state managed by the `NxList` based on whether it has any children.  This would match what we do in `NxTable.Body`.

Reply:
> I doubt we want this to be an explicit component but rather have the empty state managed by the `NxList` based on whether it has any children. This would match what we do in `NxTable.Body`.

`NxList.Empty` has been removed. Instead now it works similar to `NxTable.Body` where it displays the empty text based on the children. The empty message is displayed only if no children (besides `NxH3`) of `NxList` exist.
9c20728ac11326ee3289071ea1c0b76be6c3a745